### PR TITLE
Handle multiple polygons that are initially drawn in the drawingtool

### PR DESCRIPTION
**Pull request by royderks**

### CODE REVIEW

File: stories/src/examples/DrawToolWithMarkerClusterGroup.tsx [hunk 1]
```diff
@@ -124,27 +126,39 @@ const DrawToolWithMarkerClusterGroup: React.FC<Props> = ({
...
+        ),
+      ] as Array<PolygonType>,
     [],
   )
 
   return (
     <Map setInstance={setMapInstance} fullScreen>
       {showDrawTool &&
-        markerGroups.map(({ markers, id }) => (
+        markerGroups.map(({ markers, id }: MarkerGroup) => (
```
Comment (jonkoops) [inline stories/src/examples/DrawToolWithMarkerClusterGroup.tsx:161]: Can't TypeScript infer this type?

File: stories/src/examples/DrawToolWithMarkerClusterGroup.tsx [hunk 2]
```diff
@@ -155,21 +169,21 @@ const DrawToolWithMarkerClusterGroup: React.FC<Props> = ({
               await getMarkerGroup(layer)
               bindDistanceAndAreaToTooltip(layer)
             }}
-            onDelete={(layersInEditMode) => {
+            onDelete={(layersInEditMode: Array<ExtendedLayer>) => {
```
Comment (jonkoops) [inline stories/src/examples/DrawToolWithMarkerClusterGroup.tsx:172]: Can't TypeScript infer this type?

File: stories/src/examples/DrawToolWithMarkerClusterGroup.tsx [hunk 3]
```diff
@@ -155,21 +169,21 @@ const DrawToolWithMarkerClusterGroup: React.FC<Props> = ({
...
               const editLayerIds = layersInEditMode.map(({ id }) => id)
 
               // remove the markerGroups.
               if (markerGroupsRef.current) {
                 setMarkerGroups(
                   markerGroupsRef.current.filter(
-                    ({ id }) => !editLayerIds.includes(id),
+                    ({ id }: MarkerGroup) => !editLayerIds.includes(id),
```
Comment (jonkoops) [inline stories/src/examples/DrawToolWithMarkerClusterGroup.tsx:179]: Can't TypeScript infer this type?

File: stories/src/examples/DrawToolWithMarkerClusterGroup.tsx (hunk 1 above)
Comment (royderks) [reply stories/src/examples/DrawToolWithMarkerClusterGroup.tsx:161]: Apparently not... At least not in my IDE

File: stories/src/examples/DrawToolWithMarkerClusterGroup.tsx (hunk 3 above)
Comment (royderks) [reply stories/src/examples/DrawToolWithMarkerClusterGroup.tsx:179]: Apparently not... At least not in my IDE

File: stories/src/examples/DrawToolWithMarkerClusterGroup.tsx (hunk 2 above)
Comment (royderks) [reply stories/src/examples/DrawToolWithMarkerClusterGroup.tsx:172]: Apparently not... At least not in my IDE

**Pull request merged**